README.md

```diff
@@ -1,338 +1,58 @@
- - `sublimelinter-flake8`
- - `SublimeLinter-contrib-eslint` ([Sublime linter eslint instructions](https://github.com/roadhump/SublimeLinter-eslint#plugin-installation))
- - `babel` ([Babel instructions](https://github.com/babel/babel-sublime#setting-as-the-default-syntax))
-
-**Sublime Linting with `pyenv`**   
-To get Sublime to lint Python code using `pyenv` you must add the python version and paths and python_paths to your Sublime Linter Preferences. In **Sublime Text**, navigate to the user linter preferences:  
-
-- Sublime Preferences  -> Package Settings -> Sublime Linter -> Settings-User
-
-- Add the following (modify existing preference settings file or add this entire JSON object below if the file is blank):
-
-```json
-{
-    "user": {
-        "@python": 3.4,
-        
-        "paths": {
-            "linux": [],
-            "osx": [
-                "/Users/YOURUSERNAME/.pyenv/versions/3.4.3/bin/",
-                "/Users/YOURUSERNAME/.pyenv/versions/2.7/bin/"
-            ],
-            "windows": []
-        },
-        "python_paths": {
-            "linux": [],
-            "osx": [
-                "/Users/YOURUSERNAME/.pyenv/versions/3.4.3/bin/python3",
-                "/Users/YOURUSERNAME/.pyenv/versions/2.7/bin/python"
-            ],
-            "windows": []
-        }
-    }
-}
-```
-
- - Restart Sublime
-
-### Visual Studio Code
-
-Go to the Visual Studio Code marketplace and install these extensions:
-
- - ESLint
- - Python
- - Sass
-
-
-
-## Check versions and linting
-
-**Versions**
-
-- `python3 --version` _returns `Python 3.4.3` (or variant like  3.4.x)_
-- `node --version`  _returns `v10.15.0`  (or variant like  v6.x.y)_
-- `elasticsearch -v` _returns `Version: 1.7.6` (or variant like  Version: 1.7.x)_
-- `postgres --version` _returns `postgres (PostgreSQL) 9.3` (or variant like 9.3.x)_ 
-
-
-**Linting check**
-
-- **Python**: Open Sublime, make a change to a Python file, make an intentional syntax error (no `:` at the end an `if` evaluation). Warnings and errors should show on the left side of the line.    
-  
-  
-- **JavaScript**: Make a syntax error, or style error, in a JS file. Warnings and errors should show on the left side of the line.  
-
+ENCODE Metadata Database
+========================
+
+
+## System Installation (OSX Catlina 10.15.2)
+See [Snovault OSX System Installation][].  Snovault is a dependecy of the Encoded application.
+Installing Snovault is not a requirement to run the this application, however, the
+system installation is the same.
+
+
+## Application Installation
+For issues see [Snovault OSX App Installation][] first.
+
+1. Create a virtual env in your work directory.
+    Here we use python3 venv module.  Use venv, like conda, if you please
+    ```
+    $ cd your-work-dir
+    $ python3 -m venv encoded-venv
+    $ source encoded-venv/bin/activate
+    ```
+
+1. Clone the repo and install requirements
+    ```
+    $ cd encoded
+    # Make sure you are in the encoded-venv
```

Comment:
I think "# Make sure you are in the encoded-venv" should be mentioned once early. I believe saying it multiple times is noise.